src/plone/restapi/serializer/dxcontent.py

```diff
@@ -42,6 +42,10 @@
 @implementer(ISerializeToJson)
 @adapter(IDexterityContent, Interface)
 class SerializeToJson:
+    
+    def get_allow_discussion_value(context, request):
+    return getMultiAdapter((context, request), name="conversation_view").enabled()
```

Comment:
This function must be outside the `SerializeToJson` class. And be imported into `site.py`. Maybe it's good for the test to stay there too. So the code would look like this:

```python
def get_allow_discussion_value(context, request, result):
    if "allow_discussion" in result:
        result["allow_discussion"] = getMultiAdapter((context, request), name="conversation_view").enabled()
```